scheds/rust/scx_lavd/src/bpf/power.bpf.c

```diff
@@ -99,26 +139,82 @@ static int calc_nr_active_cpus(void)
 	 *             = (nr_cpus_onln * 1024) * (avg_sc_util / 1024)
 	 *             = nr_cpus_only * scaled_utilization
 	 */
-	req_cap = nr_cpus_onln * sys_stat.avg_sc_util;
+	return nr_cpus_onln * sys_stat.avg_sc_util;
+}
 
+static u64 get_human_readable_avg_sc_util(u64 avg_sc_util)
+{
 	/*
-	 * Fill the required compute capacity in the CPU preference order,
-	 * utilizing each CPU in a certain % (LAVD_CC_PER_CORE_UTIL or
-	 * LAVD_CC_PER_CORE_SHIFT).
+	 * avg_sc_util is the utilization assuming all the CPUs are the
+	 * fastest ones (i.e., capacity = 1024) at the highest frequency.
+	 * Hence, when all CPUs are 100% utilized, the avg_sc_util is not
+	 * 1024, it is the sum of capacities of all CPUs. Printing
```

Comment:
should this say online CPUs
```suggestion
	 * 1024, it is the sum of capacities of all online CPUs. Printing
```